### ISSUE TITLE
Inconsistent hugo version requirements for building html in README

### ISSUE THREAD

**Issue by tnbde** [darktable-org/dtdocs#709]
Hello everyone,
I tried to get the dtdocs-website running locally as described in the readme. But building the html site using hugo produces the following error:
```
$ hugo server -D --disableFastRender
Error: add site dependencies: load resources: loading templates: 
"dtdocs/themes/hugo-darktable-docs-theme/layouts/partials/css.html:4:1": 
parse failed: template: partials/css.html:4: function "css" not defined
```
Note: running the server for building the PDF works fine.

This was done using versions 0.75.1/extended and v0.73.0/extended, as mentioned in the current [readme section](https://github.com/darktable-org/dtdocs?tab=readme-ov-file#hugo). There is a typo in the readme, since v0.75.1 is mentioned in the text body and  v0.73.0 in the code block or it is outdated:

[Commit 4d1b7c6](https://github.com/pixlsus/hugo-darktable-docs-theme/commit/4d1b7c6c35de5a13844222bbc42aa22fa155ef63) implies, that newer hugo version(s) are supported now.
Hence I tried again using release of the extended version [v0.139.4](https://github.com/gohugoio/hugo/releases/tag/v0.139.4), which got the server running for me. 

So the readme may be outdated, but I am not sure about the minimal required version for hugo: Atleast v0.139.4 seems to be working.

**Comment by paperdigits** [comment]
Yeah the read me is really out dated in terms of the Hugo version. You likely want the latest Hugo version or close to it.

**Comment by tnbde** [comment]
Thanks for the info! I provided a short PR that clarifies the readme, see #710
